misc/scripts/install_vulkan_sdk_macos.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+# Download and install the Vulkan SDK.
+curl -LO "https://sdk.lunarg.com/sdk/download/latest/mac/vulkan-sdk.dmg"
+hdiutil attach "vulkansdk-macos-$VULKAN_SDK_VERSION.dmg"
+sudo "/Volumes/vulkansdk-macos-$VULKAN_SDK_VERSION/InstallVulkan.app/Contents/MacOS/InstallVulkan" \
+    --root "$HOME/VulkanSDK" --accept-licenses --default-answer --confirm-command install
```

Comment:
It's possible to use something like `﻿﻿hdiutil attach ./vulkan-sdk.dmg -mountpoint /Volumes/vulkan-sdk` to avoid version in the mount point name.

```suggestion
hdiutil attach vulkan-sdk.dmg -mountpoint /Volumes/vulkan-sdk
sudo "/Volumes/vulkan-sdk/InstallVulkan.app/Contents/MacOS/InstallVulkan" \
    --root "$HOME/VulkanSDK" --accept-licenses --default-answer --confirm-command install
```